app/assets/stylesheets/layout/_profile.scss

```diff
@@ -1,122 +1,128 @@
-@import "bootstrap";
+// @import "bootstrap";
 @import "../base/*";
 
 .profile {
-	padding-top: 35px;
-	background: #e5e3d3;
-	label {
-		font-size: 20px;
-		font-weight: normal;
-		color: #6c6c6c;
-	}
-	.form-control {
-		padding: 20px 26px;
-		font-size: 14px;
-		height: auto;
-		color: #7e7e7d;
-		background-color: #f4f4f1;
-		background-image: none;
-		border: none;
-		border-radius: 0;
-		box-shadow: none;
-		transition: none;
-	}
-	.form-control:hover {
-		box-shadow: none;
-		transition: none;
-	}
+  background: $color-ice-white;
+  padding-top: 35px;
+  label {
+    font-size: 20px;
+    font-weight: normal;
+    color: $color-dark-grey;
+  }
+  .form-control {
+    padding: 20px 26px;
```

Comment:
Properties should be ordered background-color, background-image, border, border-radius, box-shadow, color, font-size, height, padding, transition